docs/en/oas3/OAS3-PDND-WP.yaml

```
@@ -0,0 +1,299 @@
+              type: string
+              example: "cdb52532-dd94-40ef-824d-9c55b10e6bc9"
+            typ:
+              description: It MUST be set to 'JWT'.
+              type: string
+              example: "JWT"
+          required: [alg, kid, typ]
+        Payload:
+          type: object
+          properties:
+            iss:
+              description: The identifier of the e-Service.
+              type: string
+            aud:
+              description: The identifier of the Consumer.
+              type: string
+              example: "31670092-eec0-4f95-88da-e1c7ce5e4505"
+            exp:
+              description: UNIX timestamp representing the JWT expiration time.
+              type: integer
+              example: 1736846928
+            iat:
+              description: UNIX timestamp representing the JWT issuance time.
+              type: integer
+              example: 1736846688
+            jti:
+              description: Unique identifier of the JWT to prevent replay attacks.
+              type: string
+              example: "8b971b43-e990-44fa-9013-1b353bfc5a0f"
+            nbf:
+              description: UNIX timestamp representing the JWT first validity time.
+              type: string
+              example: "1736846688"
+            result:
+              type: object
+              properties:
+                revoked:
+                  type: array
+                  items:
+                    type: string
+                  description: List of Wallet Instances successfully rekoved.
+                not_found:
+                  type: array
+                  items:
+                    type: string
+                  description: List of Wallet Instances not revoked as their identifier were not found at the Wallet Provider.
+                already_revoked:
+                  type: array
+                  items:
+                    type: string
+                  description: List of Wallet Instances that were already revoked by the Wallet Provider.
+              required: [revoked, not_found, already_revoked]
+              description: >-
+                JSON object specifying which Wallet Instance was successfully revoked, which was already revoked and which was not found.
+            result_description:
+              description: Response Description.
+              type: string
+              example: The service partially process the revocation of the requested Wallet Instances as some identifiers were not found.
+          required: [iss, aud, exp, iat, jti, result, result_description]
+    notifyUserDeath:
+      required:
+        - wallet_instance_id
+      type: object
+      properties:
+        wallet_instance_ids:
+          type: array
+          items:
+            type: string
+          description: >-
+            Identifiers of the Wallet Instances associated with the User whose PID was revoked due to his/her death.
```

```suggestion
            Identifier of the Wallet Instance whose PID was revoked due to User death.
```

---

``wallet_instance_ids`` is an array of strings containing the identifiers of the Wallet Instances. In the current scenario, where we have only one Wallet Instance with a valid PID, the array will contain only one entry. Can I go back to the plural with this: 
```suggestion
            Identifiers of the Wallet Instances whose PID was revoked due to User death.
```